tools/clang/lib/SPIRV/SpirvEmitter.cpp

```diff
@@ -892,32 +892,23 @@ SpirvInstruction *SpirvEmitter::loadIfGLValue(const Expr *expr,
         const auto fromType =
             vecSize == 1 ? uintType
                          : astContext.getExtVectorType(uintType, vecSize);
```

Comment:
I am curious why we always use `getExtVectorType()` instead of `getVectorType()`. Could you please explain that?

Reply:
`getVectorType()` requires an additional argument of type `VectorKind`. See: https://github.com/microsoft/DirectXShaderCompiler/blob/298076a941bc02b1c6ab75c59bc84d4f2ecd9ea3/tools/clang/include/clang/AST/Type.h#L2701-L2708